product_customer_code/models/product_customer_code.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2012 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info@vauxoo.com
+############################################################################
+#    Coded by: Rodo (rodo@vauxoo.com),Moy (moylop260@vauxoo.com)
+############################################################################
+
+from odoo import models, fields
+
+
+class ProductCustomerCode(models.Model):
+    _name = "product.customer.code"
+    _description = "Add multiple product customer codes"
+
+    _rec_name = 'product_code'
+
+    product_code = fields.Char(
+        string='Customer Product Code',
+        required=True,
+        help="""This customer's product code will be used when searching into
```

Comment:
This is adding a bunch of whitespace into the string. Please start and end the lines as so:
```
    help="This customer's product code will be used when searching into"
         "a request for quotation."
```

Hint in a lot of IDEs, you can simply hit "Return" while in a string to do this automagically